index.html

```diff
@@ -32,5 +32,203 @@ <h1 class="heading">
+                <p class="folio-btn">
+                    <span>
+                        See this Project
+                    </span>
+                    <svg width="21" height="17" viewBox="0 0 21 17" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M16.7423 7.62613L11.4834 2.36722L12.8976 0.953003L20.5707 8.62611L12.8976 16.2992L11.4834 14.885L16.7422 9.62613H2.52881e-07L0 7.62613H16.7423Z" fill="currentColor"/>
+                    </svg>
+                </p>
+            </div>
+            <div class="folio-txt">
+                <img class="folio-img" src="./assets/folio3.png" alt="folio1">
+                <p class="folio-heading">Project name goes here</p>
+                <ul class="folio-tag">
+                    <li class="tag-item">HTML/CSS</li>
+                    <li class="tag-item">Ruby on Rails</li>
+                    <li class="tag-item">Javascript</li>
+                </ul>
+                <p class="folio-btn">
+                    <span>
+                        See this Project
+                    </span>
+                    <svg width="21" height="17" viewBox="0 0 21 17" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M16.7423 7.62613L11.4834 2.36722L12.8976 0.953003L20.5707 8.62611L12.8976 16.2992L11.4834 14.885L16.7422 9.62613H2.52881e-07L0 7.62613H16.7423Z" fill="currentColor"/>
+                    </svg>
+                </p>
+            </div>
+            <div class="folio-txt">
+                <img class="folio-img" src="./assets/folio4.png" alt="folio1">
+                <p class="folio-heading">Project name goes here</p>
+                <ul class="folio-tag">
+                    <li class="tag-item">HTML/CSS</li>
+                    <li class="tag-item">Ruby on Rails</li>
+                    <li class="tag-item">Javascript</li>
+                </ul>
+                <p class="folio-btn">
+                    <span>
+                        See this Project
+                    </span>
+                    <svg width="21" height="17" viewBox="0 0 21 17" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M16.7423 7.62613L11.4834 2.36722L12.8976 0.953003L20.5707 8.62611L12.8976 16.2992L11.4834 14.885L16.7422 9.62613H2.52881e-07L0 7.62613H16.7423Z" fill="currentColor"/>
+                    </svg>
+                </p>
+            </div>
+            <div class="folio-txt">
+                <img class="folio-img" src="./assets/folio5.png" alt="folio1">
+                <p class="folio-heading">Project name goes here</p>
+                <ul class="folio-tag">
+                    <li class="tag-item">HTML/CSS</li>
+                    <li class="tag-item">Ruby on Rails</li>
+                    <li class="tag-item">Javascript</li>
+                </ul>
+                <p class="folio-btn">
+                    <span>
+                        See this Project
```

Comment:
- [ ] See this project should be a link/ button as it will enable u get to the project . please fix that for all